rfcs/012-add-session-ssh-support/012-add-session-ssh-support.md

```diff
@@ -0,0 +1,438 @@
+- Start Date: 2023-01-20
+- Status: Proposed
+
+# Add ssh support to sessions
+
+## Summary
+
+> One paragraph explanation of the change.
+
+An important requirement which must be met by Renku is to enable users to
+log in to their Renku session via ssh; a key driver for this is to support
+working with VS Code within Renku sessions.
+
+This has been discussed within the team and an interim solution has been
+developed which is currently in operation. That solution, however, has some
+limitations; this RFC provides context, documents possible solutions and will
+be used to justify the final agreed solution.
+
+## Motivation
+
+> Why are we doing this? What use cases does it support? What is the expected
+outcome?
+
+There is a [Shape-up
```

Comment:
we've now transferred this into this repo so it can be referenced and public: https://github.com/SwissDataScienceCenter/renku-design-docs/blob/main/feature-pitches/003-ssh-into-sessions/ssh-into-sessions.md